README.md

```diff
@@ -84,6 +84,52 @@ All of the generated files can be found in the `./results` directory. Inside
 this directory will be a directory for each SRAM which contains the .lef, .lib
 and v file (as well as some intermediate files used for Cacti).
 
+### Comparison with standard SRAMs generated with OpenRAM compiler
+
+#### Generated Fakerams (Eg:- fakeram130_1024x8)
+
+<img src="https://user-images.githubusercontent.com/58559090/139503688-22503f55-b565-4e00-8f3e-af25aa50d3b1.png" width="800">

```

Comment:
These images will stop working. You should include the images in this repository and use `![fakeram130_1024x8](img/fakeram130_1024x8.png)` in your file.

Reply:
Thanks for pointing that out! 
I will make that change.